manifests/a/AVASTSoftware/AvastSecureLine/5.2.438.0/AVASTSoftware.AvastSecureLine.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: AVASTSoftware.AvastSecureLine
+PackageVersion: 5.2.438.0
+PackageLocale: en-US
+Publisher: AVAST Software
+PackageName: Avast SecureLine
+License: Subscription License
+Copyright: Copyright © 2014 AVAST Software
+ShortDescription: Avast SecureLine Setup

```

Comment:
Please provide a better description of what this package is, does, or is used for